bin/test_travis.sh

```diff
@@ -97,8 +97,3 @@ fi
 
 echo "Checking whether all header files are installed:"
 python $SOURCE_DIR/symengine/utilities/tests/test_make_install.py $our_install_dir/include/symengine/ $SOURCE_DIR/symengine
-
-if [[ "${TEST_CODE_QUALITY}" == "yes" ]]; then
-    python $SOURCE_DIR/symengine/utilities/tests/test_code_quality.py
-fi
```

Comment:
That's a good point. My original idea was to run tests so that white space errors don't prevent tests from running, but we can fail early as well.